rtrlib/rtr_mgr.c

```diff
@@ -293,7 +293,7 @@ int rtr_mgr_init(struct rtr_mgr_config **config_out,
 	struct pfx_table *pfxt = NULL;
 	struct spki_table *spki_table = NULL;
 	struct rtr_mgr_config *config = NULL;
-	*config_out = NULL;
```

Comment:
mhm, okay I'll re-add that - to be double sure 😉 

IMHO it should be the responsibility of the calling function to ensure correct usage and initialization of `config_out`.